profile/rwahs.xml

```diff
@@ -7488,6 +7488,69 @@
       <actions>
       </actions>
     </role>
+    <role code="cataloguer">
+      <name>Cataloguers</name>
+      <description>Confers actions needed by those editing catalogue entries</description>
+      <actions>
```

Comment:
Should there be tests that check at least the number, if not the list, of `actions` per `role`?